web/src/HeaderBar.stories.tsx

```diff
@@ -46,3 +54,49 @@ export const UpgradeAvailable = () => {
   status!.versionSettings = { checkUpdates: true }
   return <HeaderBar view={view} currentPage={AnalyticsType.Detail} />
 }
+
+// TODO (lizz): Use HeaderBar component instead of GlobalNav
+// when design & implementation are finalized
```

Comment:
Just checking that this is still a valid `TODO` (i.e. you're not ready to move to `HeaderBar` here yet)

Reply:
yup! i'll do this when we're ready to ship the cluster status info to users. right now, the cluster info is only passed to the `GlobalNav` component in storybook.